fmriprep/config.py

```diff
@@ -167,15 +169,16 @@ def __init__(self):
         raise RuntimeError('Configuration type is not instantiable.')
 
     @classmethod
-    def load(cls, settings, init=True):
+    def load(cls, settings, init=True, ignore=None):
         """Store settings from a dictionary."""
         for k, v in settings.items():
+            if ignore and k in ignore:
+                continue
             if v is None:
                 continue
             if k in cls._paths:
                 setattr(cls, k, Path(v).absolute())
-                continue
-            if hasattr(cls, k):
+            elif hasattr(cls, k):
```

Comment:
what's the use case for this change (if -> elif)

Reply:
it was just to remove the continue